### DISCUSSION TITLE
Kernel update for linux-qoriq in scarthgap

### DISCUSSION THREAD

**Discussion by bcoprescu** [Freescale/meta-freescale#1873] [General]
Hi,

Is there any plan to brig linux-qoriq to 6.x in scarthgap?

Thx,
Bogdan

**Comment by angolini** [comment]
@otavio, when you are available, could you please help send this question to someone who can answer it?

**Comment by bcoprescu** [comment]
Hi @otavio, @angolini,

Any new info on this?

Thx,
Bogdan

**Reply by otavio** [reply]
I spoke to NXP yesterday and they confirmed that someone is going to take a look at this.
